screens/population-dynamics/index.js

```diff
@@ -16,7 +18,7 @@ export function PopulationDynamics() {
 
   return (
     <section
-      id="population-dynamics"
+      id={t("population_dynamics.id")}
```

Comment:
I understand the translated id is so the url anchor is translated too right?

Reply:
Exactly, to allow people to link to https://wildanimalsuffering.org/es/#dinámica-poblacional or something like that :P